src/components/ToDoList/ToDoList.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from "react";
+import { StyledToDoList } from "./toDoList.style";
+import ToDo from "../ToDo/ToDo";
+import ToDoInput from "../ToDoInput";
+
+interface IToDos {
+  id: number;
+  isComplete: boolean;
+}
+
+const ToDoList = () => {
+  const [toDos, setToDos] = useState<IToDos[]>([]);

```

Comment:
I think the initial state should be the todo object:
`[{
isComplete: false,
text: ''
}]`